client/components/domains/registrant-extra-info/uk-form.jsx

```diff
@@ -30,6 +30,8 @@ export class RegistrantExtraInfoUkForm extends React.PureComponent {
 	static propTypes = {
 		contactDetails: PropTypes.object.isRequired,
 		ccTldDetails: PropTypes.object.isRequired,
+		onContactDetailsChange: PropTypes.func,
+		contactDetailsValidationErrors: PropTypes.object,
```

Comment:
Same as the other files, I'd suggest adding `updateContactDetailsCache` as a required proptype also as long as you're adjacent to it.